picard/script/serializer.py

```diff
@@ -245,7 +245,7 @@ def export_script(self, parent=None):
             with open(filename, 'w', encoding='utf-8') as o_file:
                 o_file.write(script_text)
         except OSError as error:
-            raise ScriptSerializerExportError(format=FILE_ERROR_EXPORT, filename=filename, error_msg=error.strerror)
+            raise ScriptSerializerExportError(format=FILE_ERROR_EXPORT, filename=filename, error_msg=error.strerror) from None
```

Comment:
Should this better raise `from error`, to not hide the file access issue? Could be important for debugging.